### ISSUE TITLE
ACR spatial resolution

### ISSUE THREAD

**Issue by elizaGSTT** [GSTT-CSC/hazen#369]
@YassineAzma i am not sure i get the spatial resolution outcomes. shouldn't those values be up to 1 since the hole size in the insert is 1mm? most values come out quite large.

<img width="957" alt="Screenshot 2023-08-02 at 15 02 33" src="https://github.com/GSTT-CSC/hazen/assets/93722461/af89fe1e-4a2b-446c-8983-042896106f67">

**Comment by YassineAzma** [comment]
Does your data have a slanted edge?

**Comment by elizaGSTT** [comment]
I guess the two pairs of the boxes with the holes are slightly slanted.. but i presume this is not what's measured? qualitatively we just look at how well these are separated for this test

<img width="582" alt="image" src="https://github.com/GSTT-CSC/hazen/assets/93722461/c4f4be94-d95c-4e19-8763-b9edc7d33a56">

**Comment by YassineAzma** [comment]
The MTF test for the ACR phantom currently uses the edge where the slice thickness inserts are to measure. 

![image](https://github.com/GSTT-CSC/hazen/assets/96583432/729ed44d-27e0-4722-852f-75dce9b4694f)
![image](https://github.com/GSTT-CSC/hazen/assets/96583432/2815a0cf-1703-4e43-b21e-ad72f77a3358)

The test is described in Section 3 of this paper: https://opg.optica.org/oe/fulltext.cfm?uri=oe-22-5-6040&id=281325

If you'd like an option to utilise the old algorithm that was implemented for the MagNet phantom, raise an issue.

Just to reiterate, the ACR is not intended to be used to measure MTF. I don't have a suitable phantom or any time to validate this. 

When performing this test for the MagNet phantom, did you receive values equal to the nominal spatial resolution?

**Comment by elizaGSTT** [comment]
So that means that we need to rotate the phantom for this test to work?

**Comment by YassineAzma** [comment]
Yes, this was discussed during meetings last year with Juliet. I don't recall if you were there. I'll go through the code of this function sometime this month, just to ensure that everything is working as it should be.

**Comment by elizaGSTT** [comment]
I must have been there during most meetings, but I don’t believe we ever came to a conclusion. And I think because sometimes it gives sensible readings I didn’t pay too much attention but I thought I’d bring it up again.

**Comment by YassineAzma** [comment]
What are sensible readings? I'm not sure what the relationship between MTF50, which is what is being reported, and visual perception is. This is why I'm asking what type of results you get with the MagNet phantom. Were they comparable to the in-plane resolution?

**Comment by elizaGSTT** [comment]
When its close to 1 I always presumed it meant closer to 1mm which is what we are looking at qualitatively but you’re using the slanted edge method so I’m not entirely sure of what value we should get for that. I definitely need to ask David again cause I think he is the only one understanding this and he is currently on leave.
